entities/entities/spawned_shipment/init.lua

```diff
@@ -130,10 +130,16 @@ function ENT:SpawnItem()
     local class = CustomShipments[contents].entity
     local model = CustomShipments[contents].model
 
+    local defaultClip
+    local wep_tbl = weapons.Get(class)
```

Comment:
With your history of micro-optimisations, I expected this to be `weapons.GetStored`.

Reply:
weapons.GetStored doesn't inherit settings from superclasses. Really, the weapons/scripted_ents libraries should have some kind of GetInherited function that caches the generated table.

Reply:
Better use CPU there to micro optimise memory usage

Reply:
This is the type of thing that should be documented on the wiki but is not. The wiki recommends using `weapons.GetStored` where possible.